integration/js/BackgroundReplacementTest.js

```diff
@@ -0,0 +1,41 @@
+const {ClickVideoButton, ClickBackgroundReplacementButton} = require('./steps');
+const {VideoBackgroundReplacementCheck, LocalVideoCheck, RemoteVideoCheck} = require('./checks');
+const {AppPage} = require('./pages/AppPage');
+const {TestUtils} = require('kite-common');
+const {SdkTestUtils} = require('./utils/SdkTestUtils');
+const SdkBaseTest = require('./utils/SdkBaseTest');
+const { v4: uuidv4 } = require('uuid');
+const {Window} = require('./utils/Window');
+
+class BackgroundReplacementTest extends SdkBaseTest {
```

Comment:
Can we reuse the blur test? These are identical.

Reply:
They are very close, but slightly different. IMO it is better to have a little repeated code, but remove coupling between the tests. That way we can easily extend with out breaking each other.

In this case, the steps are slightly different. It uses blur button and blur check vs. replacement button and replacement check.

Reply:
Ok I am fine with keeping the test separate. However, I still dont see why we need 2 files for the check.

Reply:
I have consolidated logic and removed code duplication between the tests.